mis_builder/static/src/components/mis_report_widget.css

```diff
@@ -1,3 +1,22 @@
+.oe_mis_builder_content {
+    max-width: 98vw;
+}
```

Comment:
Is this change necessary for the sticky header? Or is it a remnant of the horizontal scroll attempt? If the latter, let's remove it.